server/resolvers.js

```diff
@@ -0,0 +1,23 @@
+const { gql } = require("apollo-server-express");
+
+exports.resolvers = {
+  Mutation: {
+    sampleRevisableMutation: async (_, { smileSampleId, revisable }) => {
+      return updateSamples;
+    }
+  }
+};
+
+exports.sampleRevisableMutationQuery = gql`
```

Comment:
this should actually accept all of the sample properties which the user is updating.  perhaps even just all the sample properties.   it doesn't know about revisable.